bsp/stm32/stm32f103-atk-warshipv3/board/Kconfig

```diff
@@ -34,6 +34,10 @@ menu "Onboard Peripheral Drivers"
         select BSP_USING_ADC1
         default n 
 
+    config BSP_USING_SRAM
+        bool "Enable SRAM"
```

Comment:
Could you please provide more information about this SRAM? Which SRAM?

Reply:
OK，I've added information!